lib/twitter.js

```diff
@@ -250,6 +250,8 @@ Twitter.prototype._buildReqOpts = function (method, path, params, isStreaming, c
     if (path.indexOf('media/') !== -1) {
       // For media/upload, use a different endpoint.
       reqOpts.url = endpoints.MEDIA_UPLOAD + path + '.json';
+    } else if (self.config.version) {
```

Comment:
Is there something wrong with the indentation?

Reply:
It looks fine to me 👀